@re-/model/demos/generateCode/markup.ts

```diff
@@ -1,34 +1,30 @@
-import { fromHere, readFile, writeFile } from "@re-/node"
 import { remark } from "remark"
 import parseMarkdown from "remark-parse"
 import { regex } from "./regex.js"
+import { fromHere, readFile, writeFile } from "@re-/node"
 
 const README_PATH = fromHere("../../README.md")
 const CODE_TAG = "code"
 const GENERATED = "***GENERATED***"
 
-const visitor = (root: any, mapData: any) => {
-    for (const child in root.children) {
-        const node = root.children[child]
-        if (
-            node.type === CODE_TAG &&
-            node.meta &&
-            regex.markdownTagMatch.test(node.meta)
-        ) {
-            const metaData = node.meta.replace(GENERATED, "").trim()
-            if (mapData[metaData]) {
-                node.value = mapData[metaData]
-                if (!node.meta.includes(GENERATED)) {
-                    node.meta = `${GENERATED} ${node.meta}`
+const generateCode =
+    ({ mapData }: any) =>
```

Comment:
Since these types are our own, we can probably do a little better than `any` here. E.g., we should definitely have some sort of type representing the object that `mapData` is attached to, even if mapData itself is typed as `any`. 

Same thing for `tree`, and ideally maybe we can also use a variable name that's a bit more descriptive.